README.md

```diff
@@ -29,21 +29,30 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. Describe the biggest difference between `.forEach` & `.map`.
+   forEach Is a built-in iterator that performs for each 'item'/'value' a callback's body. It return: undefined.
+   .map() Is an iterator that performs a given funtionality passed as a callback on each 'item'/'value'. This method return a new array with the values returned by the callback.
 
 2. What is the difference between a function and a method?
+   Functions are objects taht are no tide to any other object, and methods are tied to another Object.
 
 3. What is closure?
+   A closure is what a function can have access to.
 
 4. Describe the four rules of the 'this' keyword.
+   A 'new' brand created Object. At Objects instantiation when the 'new' keyword is used.
+   An Object Explicitly when it invokes using either: .call, .bind or .apply
+   An Object Implicitly when it is invoked from this Object as a method with the 'dot' notations.
+   Global object (The window in browsers) when neither of the other cases is applied.
 
 5. Why do we need super() in an extended class?
+   Is the keyword that allow us to 'refer'/'call' the class from which is inherit.
```

Comment:
`The super() is used to access and call functions on an object's parent. When used in a constructor, the super keyword appears alone and must be used before the this keyword is used.`